### PR TITLE
FormFooter component

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/@next/components/molecules/FormFooter/FormFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { Button, ButtonLink } from "@components/atoms";
+import * as S from "./styles";
+import { IButtonProps, IProps } from "./types";
+
+const getBtnAction = (btn: IButtonProps) =>
+  btn.action && { onClick: btn.action };
+
+const renderCancelBtn = (cancelBtn?: IButtonProps) =>
+  cancelBtn && (
+    <ButtonLink {...getBtnAction(cancelBtn)} type="button" color="secondary">
```
Comment (jwm0) [inline src/@next/components/molecules/FormFooter/FormFooter.tsx]: ButtonLink doesn't seem to have any hover/active styles - is this how it's supposed to be?

Comment (AlicjaSzu) [reply src/@next/components/molecules/FormFooter/FormFooter.tsx]: it should have hover state...i'll fix it soon

File: src/@next/components/atoms/ButtonLink/__snapshots__/stories.storyshot [hunk 2]
```diff
@@ -33,6 +33,12 @@ exports[`Storyshots @components/atoms/ButtonLink Base 1`] = `
   -webkit-text-decoration: underline;
   text-decoration: underline;
   padding: 0;
+  -webkit-transition: color 0.3s ease;
```
Comment (jwm0) [inline src/@next/components/atoms/ButtonLink/__snapshots__/stories.storyshot]: no need to use webkit and other custom browser-engine specific CSS styles, styled-components auto-applies them in runtime

Comment (AlicjaSzu) [reply src/@next/components/atoms/ButtonLink/__snapshots__/stories.storyshot]: there is no -webkit prefix added in styles, it's only in snapshots

Comment (jwm0) [reply src/@next/components/atoms/ButtonLink/__snapshots__/stories.storyshot]: 🤦‍♂ right

**Pull request merged**